README.md

```diff
@@ -1,44 +1,57 @@
-NOTE: This is a WIP - I plan to write proper documentation and make a formal release soon.
+# graphql_client
 
-Derive Rust code to safely interact with queries written in the GraphQL query language.
+![CI Status](https://img.shields.io/travis/tomhoule/graphql-client.svg)
+![docs](https://docs.rs/mio/badge.svg)
+![crates.io](https://img.shields.io/crates/d/graphql-client.svg)
+![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
 
-This library does not provide any networking, caching or other client functionality, it is just meant to make it easy to interact with a GraphQL query and the corresponding response in a strongly typed way. Building a client can be as simple as this:
+Derive Rust code to safely and ergonomically manipulate GraphQL queries.
+
+This library does not provide any networking, caching or other client functionality yet, it is just meant to make it easy to interact with a GraphQL query and the corresponding response in a strongly typed way. Making a request can be as simple as this:
```

Comment:
I'd remove the `yet`, it leaves the intention of the library and its roadmap unclear.

Reply:
This paragraph has the purpose of saying what the library does/does not, but you also introduce the recipe with the last sentence. The last sentence should be in a new line and in bold (IMO).

Reply:
I think the purpose of the library (i.e. what it _does_) should go closer to the tagline "Derive Rust code..." and the "what it _doesn't_ do" should go elsewhere, maybe after the features section, and it should be visually different (italics or something).

Reply:
The fact that it doesn't provide networking, caching or other functionalities is not part of the "purpose" of the library (apparently) so I think it should be mentioned a lot less prominently.